test/e2e/storage/persistent_volumes-local.go

```diff
@@ -590,14 +591,15 @@ var _ = utils.SIGDescribe("PersistentVolumes-local ", func() {
 						numFinished++
 						framework.Logf("%v/%v pods finished", numFinished, totalPods)
 					case v1.PodFailed:
+						fallthrough
```

Comment:
it makes sense that `PodFailed` is an unrecoverable error, but actually is `PodUnknown` unrecoverable? Should that case actually just continue through Polling loop instead of throwing an error?

Reply:
In the existing code, `PodUnknown` is unrecoverable as we return an error, while `PodFailed` is just skipped and we get to the return `nil` on line 600.

Either this was intended to be a fallthrough but the original author wrote it the wrong way, or it is intentionally a no-op and we should add a comment. I'll be safe and keep the current logic.